### PR TITLE
Documentation Application Profiles and Workflow for Domain-specific Extensions

### PR DESCRIPTION
**Pull request by JohannesLipp**
Part of the results of T60, refers to #504  #540 and others

### CODE REVIEW

**Review by liam-tirpitz — CHANGES_REQUESTED** [review]

**Comment by JohannesLipp** [comment]
> Both pdf files contain comments and questions in red. Please check, if the files should contain these comments before merging.

Good point!

@lcomet please update both PDF files for clarity, thanks! That is:

- Replace the single comment in the ExtendSHACLshapes with "No changes"
- Remove/replace all comments in the other PDF file, as we dropped or implemented all these anyway.

File: evaluation_external/README.md [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+                           sh:path ap15:operator;
+                           sh:minCount 0;
+                           sh:maxCount 1;
+                       ] .
+
+```
+
+Now we see that the properties `dct:title`, `dct:description`, `dct:issued`, and `ap15@serialNumber` are mandatory, as the shapes indicates that the value for the `sh:minCount` is 1.
+
+Another possibility is to use the existent definitions in the **IDS Information model** and we want to tighten the restrictions, for example for the properties `dcat:mediaType`, `ids:representationStandard `, and `ids:instsance`.
```
[the remark this answers is not in the document]
Comment (JohannesLipp) [reply evaluation_external/README.md:286]:
```suggestion
Another possibility is to take existing definitions from the **IDS Information model** and tighten their restrictions, for example for the properties `dcat:mediaType`, `ids:representationStandard `, and `ids:instsance`.
```

**Review by liam-tirpitz — CHANGES_REQUESTED** [review]

**Review by liam-tirpitz — CHANGES_REQUESTED** [review]
The ,DS-Store file should not be committed to the repository. Please delete it.

**Review by liam-tirpitz — APPROVED** [review]

**Comment by JohannesLipp** [comment]
> The ,DS-Store file should not be committed to the repository. Please delete it.

done, thanks for the hint!

**Pull request merged**